src/main/scala/nl.knaw.dans.easy.deposit/docs/DatasetMetadata.scala

```diff
@@ -221,7 +198,7 @@ object DatasetMetadata {
                       url: Option[String],
                       title: Option[String],
                      ) extends RelationType {
-    require(isValid, "Relation needs at least one of (title | url) got: ${toJson(this)}")
+    require(isValid, s"Relation needs at least one of (title | url) got: ${toJson(this)}")
 
     def isValid: Boolean = {
```

Comment:
this method is only used in the `require` above. Are you planning to use it in other places as well? Otherwise inlining this check wouldn't be a bad idea, IMO.